pegasus/sites.v3/code.org/views/dance_teacher_resources.haml

```diff
@@ -5,14 +5,14 @@
     {
       title: I18n.t(:hoc2018_dance_teacher_resources_title),
       description: I18n.t(:hoc2018_dance_teacher_resources_tile1_description),
-      image: "/images/animated-examples/cutesprite.gif",
+      image: "/images/dance-hoc/DanceParty_dance.gif",
       button_text: I18n.t(:hoc2018_dance_teacher_resources_tile1_button),
       link: CDO.studio_url('/s/dance')
     },
     {
       title: I18n.t(:hoc2018_dance_teacher_resources_tile2_title),
       description: I18n.t(:hoc2018_dance_teacher_resources_tile2_description),
-      image: "/images/fill-234x300/dance_unplugged.gif",
+      image: "/images/dance-hoc/DancePartyUnplugged_Portrait.gif",
```

Comment:
Is this a gif?

Reply:
I thought the robot was just a static image? If so, it should probably be a png

Reply:
@Erin007 Thanks good catch, it's a png